swap/src/cli/command.rs

```diff
@@ -72,6 +73,13 @@ pub enum Command {
     },
 }
 
+fn parse_monero_address(s: &str) -> Result<monero::Address> {
+    monero::Address::from_str(s).context(format!(
```

Comment:
You will want to use `with_context` as soon as you use `format!` otherwise you are incurring an unnecessary allocation in case the parsing succeeds and the error is not actually needed.